cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1252,7 +1211,7 @@ private BatchRunnableInstances getProgramInstances(BatchRunnable runnable, Appli
    * @throws BadRequestException if program type is invalid
    * @throws NotFoundException if the application to which this program belongs was not found
    */
-  private StatusMap getStatus(final Id.Program id) throws BadRequestException, NotFoundException {
+  private ProgramStatus getStatus(final Id.Program id) throws BadRequestException, NotFoundException {
```

Comment:
its confusing that we have this and the `getProgramStatus` methods. Can we combine them, or is there some reason they are separate?